src/provider.c

```diff
@@ -916,9 +925,9 @@ static CK_RV operations_init(P11PROV_CTX *ctx)
                              DIGEST_MECHS,
                              CKM_EDDSA,
 #if SKEY_SUPPORT == 1
-                             AES_MECHS
+                             AES_MECHS,
 #endif
-    };
+                             HMAC_MECHS };
```

Comment:
nit: I think if you add the trailing comma here, the clang will land the closing brace on the newline, which will make diffs of adding more elements easier to read.